docker/Dockerfile.nvidia

```diff
@@ -178,7 +167,7 @@ RUN mkdir -p /app && \
     
 # Change to the app user.
 USER app
-WORKDIR /app
+WORKDIR /app/devito
```

Comment:
I guess we are not anymore backwards compatible with any people that will be developing or running scripts upon docker installation?

Reply:
We are the root dir is different but the structure is the same